resteasy-wadl-undertow-connector/src/main/java/org/jboss/resteasy/wadl/WadlUndertowConnector.java

```diff
@@ -30,11 +30,11 @@ public UndertowJaxrsServer deployToServer(UndertowJaxrsServer server, Class<? ex
 
       DeploymentInfo di = server.undertowDeployment(deployment);
 
-      ServletInfo resteasyWadlServlet = servlet("ResteasyWadlServlet", ResteasyWadlServlet.class)
-            .setAsyncSupported(false)
-            .setLoadOnStartup(1)
-            .addMapping("/application.xml");
-      di.addServlet(resteasyWadlServlet);
+//      ServletInfo resteasyWadlServlet = servlet("ResteasyWadlServlet", ResteasyWadlServlet.class)
+//            .setAsyncSupported(false)
+//            .setLoadOnStartup(1)
+//            .addMapping("/application.xml");
+//      di.addServlet(resteasyWadlServlet);
```

Comment:
Can we please get rid of this and other commented out stuff in this PR please?

Reply:
yes sir!